backend/storj/fs.go

```diff
@@ -56,61 +58,65 @@ func init() {
 
 			access, err := uplink.RequestAccessWithPassphrase(context.TODO(), satellite, apiKey, passphrase)
 			if err != nil {
-				fs.Errorf(nil, "Couldn't create access grant: %v", err)
-				return
+				log.Fatalf("Couldn't create access grant: %v", err)
 			}
 
 			serialziedAccess, err := access.Serialize()
 			if err != nil {
-				fs.Errorf(nil, "Couldn't serialize access grant: %v", err)
-				return
+				log.Fatalf("Couldn't serialize access grant: %v", err)
 			}
 
-			configMapper.Set("access", serialziedAccess)
+			configMapper.Set("access_grant", serialziedAccess)
 		},
 		Options: []fs.Option{
 			{
-				Name: fs.ConfigProvider,
-				Help: "Choose your configuration base.",
+				Name:     fs.ConfigProvider,
+				Help:     "Choose an authentication method.",
+				Required: true,
+				Default:  "existing",
 				Examples: []fs.OptionExample{{
-					Value: "Existing Access",
-					Help:  "Use existing access grant to make configuration",
+					Value: "existing",
+					Help:  "Use an existing access grant.",
 				}, {
-					Value: "New Access",
-					Help:  "Create access grant with satellite address, API key and passphrase.",
+					Value: "new",
+					Help:  "Create a new access grant from satellite address, API key, and passphrase.",
 				},
 				}},
 			{
-				Name:     "access",
+				Name:     "access_grant",
```

Comment:
Is this desirable? We don't use the long name usually in code or config in our other tools.

Reply:
We did not use it because most of the code was done before we decide do use access grant. Here its only longer param name and entry in config file.